test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -17,8 +19,12 @@ describe('Visual graph screen validation', () => {
     });
 
     it('Test notification when autocomplete is disabled', () => {
```

Comment:
_Could_ be moved in context('When autocomplete is disabled') and the rest in similar context. The idea is to `enableAutocomplete` only where needed without having to visit multiple pages

Reply:
It's only this specific test where autocomplete is disabled. And to be honest I didn't manage to find out why it started failing. It used to pass on master. For some reason autocomplete disabling through the cypress custom command yelds successful autocomplete disabling but the test fails like it is not disabled.